src/lib/utils/search.ts

```diff
@@ -0,0 +1,11 @@
+import { APP_URL } from "@/config/config";
+
```

Comment:
I think I put the user-specific search URLs in `api/accounts.ts`. Should probably move those here so they're all in one place.